recipes-graphics/raspidmx/raspidmx_git.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "A set of DispmanX examples"
```

Comment:
Let's use the same description in github as it clearly states the fact that it is RPi specific: Some examples using the DispmanX API on the Raspberry Pi .

Reply:
Sounds good.